frontend/packages/citizen-frontend/src/decisions/shared.tsx

```diff
@@ -23,6 +38,14 @@ export const statusIcon = {
   REJECTED: {
     icon: faTimes,
     color: greyscale.lighter
+  },
+  CREATED: {
+    icon: faGavel,
```

Comment:
This should be `faFile`.

Reply:
Actually in the UX spec it is a gray gavel icon, but of course that is wrong